### PR TITLE
Lepidopter/ubuntu screenshots

### PR DESCRIPTION
**Pull request by agrabeli**
I have edited the Lepidopter guides and have added some screenshots for the Linux version. Please check this carefully before merging. :)

### CODE REVIEW

File: content/install/lepidopter.md [hunk 1]
```diff
@@ -142,202 +229,212 @@ If you have downloaded the [xz compressed image]
...
+**Note:** The SD card can show up more than once in the output of df: in fact it
+will if you have previously written a Raspberry Pi image to this SD card,
+because the Raspberry Pi SD images have more than one partition.
+
+**Step 6.** Now that you've noted what the device name is, you need to unmount
+it so that files can't be read or written to the SD card while you are copying
+over the SD image. So run the command below, replacing "/dev/sdd1" with whatever
+your SD card's device name is (including the partition number):
+
+    ```     umount /dev/sdd1     ```
```
Comment (anadahz) [inline content/install/lepidopter.md]: This breaks the formating a bit it should be in 3 lines instead.

File: content/install/lepidopter.md [hunk 2]
```diff
@@ -142,202 +229,212 @@ If you have downloaded the [xz compressed image]
...
+
+**Step 7.** In the terminal write the image to the card with this command,
+making sure you replace the input file if= argument with the path to your .img
+file, and the "/dev/sdd" in the output file of= argument with the right device
+name (this is very **important:** you will lose all data on the hard drive on
+your computer if you get the wrong device name). Make sure the device name is
+the name of the whole SD card as described above, not just a partition of it
+(for example, sdd, not sdds1 or sddp1, or mmcblk0 not mmcblk0p1).
+
+    ```     dd bs=4M if=~/lepidopter-alpha-armel.img of=/dev/sdd     ```
```
Comment (anadahz) [inline content/install/lepidopter.md]: Same as in https://github.com/TheTorProject/ooni-web/pull/65/files#r71989995 it should be in 3 lines.

File: content/install/lepidopter.md [hunk 3]
```diff
@@ -142,202 +229,212 @@ If you have downloaded the [xz compressed image]
...
+card with only a few simple clicks.
+
+**Step 4.** Insert your SD card into your SD card reader, and subsequently connect
+your SD card reader to your laptop/desktop.
+
+**Step 5.** Start ApplePi-Baker.
+
+**Step 6.** Select the Lepidopter IMG file in the Restore Backup section. 
+
+**Step 7. Select your SD card reader (in the Pi-Crust section) as a location to
```
Comment (anadahz) [inline content/install/lepidopter.md]: s/`**Step 7.`/`**Step 7.**`

File: content/install/lepidopter.md [hunk 4]
```diff
@@ -1,27 +1,79 @@
...
+repository](https://github.com/TheTorProject/lepidopter). 
+
+Some compact documentation on how to install, build, copy to an SD card and run
 Lepidopter in QEMU is described
 [here](https://github.com/TheTorProject/lepidopter/blob/master/README.md).
 
-## Download Lepidopter Raspberry Pi image
-Lepidopter Raspberry Pi image is compatible with all types of Raspberry Pi
-(tested: B+, RPi 1 model B, RPi 2 model B).
+##About Lepidopter
```
Comment (anadahz) [inline content/install/lepidopter.md]: s/`##About Lepidopter`/`## About Lepidopter`/

File: content/install/lepidopter.md [hunk 5]
```diff
@@ -1,27 +1,79 @@
...
+network router or switch that could provide internet connectivity. Typically the
+"device" that enables you to access the internet!
+
+* **Power supply:** Use a 5V micro USB power supply to power your Raspberry Pi.
+Be careful that whatever power supply you use outputs at least 5V; insufficient
+power will cause your Pi to behave in strange ways.
+
+If your laptop does not have an SD card slot, then ensure you have an SD card reader.
+
+# Downloading the verifying the Lepidopter Raspberry Pi image
```
Comment (anadahz) [inline content/install/lepidopter.md]: This should be:
`Downloading and verifying the Lepidopter Raspberry Pi image`

File: content/install/lepidopter.md [hunk 6]
```diff
@@ -348,25 +445,26 @@ with a link to instructions (also translated).
...
+If you get an access denied error, make sure to close all explorer windows or
+folders open for the device, then try re-plugging the SD card.
 
 # Sources
 
 * [RPi Easy SD Card Setup](http://elinux.org/RPi_Easy_SD_Card_Setup)
 * [Raspberry Pi Documentation](https://www.raspberrypi.org/documentation/)
 * [On Digital Signatures and Key Verification]
-(https://www.qubes-os.org/doc/verifying-signatures/)
+(https://www.qubes-os.org/doc/verifying-signatures/)
```
Comment (anadahz) [inline content/install/lepidopter.md]: This change is not needed, perhaps an extra leading space?

**Comment by anadahz** [comment]
Nice work @agrabeli apart from my comments the PR is good and should be merged.
Note that we should be consistent and either use burn or write/copy the image to the SD card throughout the document.

Thank you for working on this!

**Comment by agrabeli** [comment]
Thanks for the feedback @anadahz! I've edited accordingly, and I've tried to ensure that the word "copy" is consistent throughout the document (I chose "copy" because I think the average user probably understands that better than the word "burn", which could be misleading).

**Comment by anadahz** [comment]
Looks good merging!

**Pull request merged**